src/views/Rules/ListRules/index.jsx

```diff
@@ -433,6 +433,25 @@ function ListRules(props) {
 
         // diff viewer + marginTop
         height += diffedProperties.length * diffRowHeight + theme.spacing(1);
+
+        if (Object.keys(rule.scheduledChange.required_signoffs).length > 0) {
+          const requiredRoles = Object.keys(
+            rule.scheduledChange.required_signoffs
+          ).length;
+          const nSignoffs = Object.keys(rule.scheduledChange.signoffs).length;
+          // Required Roles and Signoffs are beside one another, so we only
+          // need to account for the one with the most items.
+          const signoffRows = Math.max(requiredRoles, nSignoffs);
+
+          // Padding above the summary
+          height += theme.spacing(2);
+
+          // The "Requires Signoff From" title and the padding beneath it
+          height += body1TextHeight() + theme.spacing(0.5);
+
+          // Space for however many rows exist.
+          height += signoffRows * (body2TextHeight() + theme.spacing(1));
```

Comment:
Rows don't seem to have margins between them so we could probably remove `theme.spacing(1)`.